lib/bulbs-elements/settings.js

```diff
@@ -0,0 +1,31 @@
+let settings = {
+  base: {
+    BULBS_ELEMENTS_ONIONSTUDIOS_GA_ID: 'UA-223393-14',
+    BULBS_ELEMENTS_COMSCORE_ID: '6036328',
+    RESP: 'vmap1',
+  },
+  development: {
+    FREEWHEEL_NETWORK_ID: '111976',
```

Comment:
my logic behind it was that they were the same for each site, so we wouldn't have to repeat them. And I just felt weird about putting them on the window.